File: phase_gate1/sum_of_user_input.py
input_numbers = int(input("Enter number between 0 and 1000: "))
sum_of_userinput = 0


while input_numbers > 0:
    sum_of_userinput += input_numbers % 10  
    input_numbers //= 10         

print("The sum of the numbers imputed is :", sum_of_userinput)

File: phase_gate1/java_version/SumOfUserInput.java
import java.util.Scanner;
public class SumOfUserInput{
	public static void main(String... arg){
		Scanner sc = new Scanner(System.in);

int sumOfUserinput = 0;
System.out.print("Enter number between 0 and 1000: ");
int numbers = sc.nextInt(); 

while (numbers > 0) {
	sumOfUserinput += numbers % 10;
	numbers = numbers / 10;
	}       

System.out.print("The sum of the numbers imputed is :" + sumOfUserinput);

	}
}